### PR TITLE
feat(package): `parse-dont-validate`

### PR DESCRIPTION
**Pull request by GervinFung**
This PR is to address the issue of #1007

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "json-decoder": "1.4.0",
     "myzod": "1.8.8",
     "ok-computer": "1.0.4",
+    "parse-dont-validate": "^4.0.0",
```
Comment (moltar) [inline package.json:50]: Please use pinned version

**Review by moltar — CHANGES_REQUESTED** [review]
Also looks like this adds a yarn lock.

File: package.json (hunk 1 above)
Comment (GervinFung) [reply package.json:50]: Sure

**Comment by GervinFung** [comment]
> Also looks like this adds a yarn lock.

Ops my bad

**Comment by moltar** [comment]
There's a test failure:

> all cases must have been imported in tests

```
  Expected: 34
    Received: 35

      48 |   });
      49 |
    > 50 |   expect(registeredCases.size).toEqual(cases.length);
         |                                ^
      51 | });
      52 |
      53 | getRegisteredBenchmarks().forEach(([benchmarkId, benchmarkCases]) => {

```

**Comment by GervinFung** [comment]
> There's a test failure:
> 
> > all cases must have been imported in tests
> 
> ```
>   Expected: 34
>     Received: 35
> 
>       48 |   });
>       49 |
>     > 50 |   expect(registeredCases.size).toEqual(cases.length);
>          |                                ^
>       51 | });
>       52 |
>       53 | getRegisteredBenchmarks().forEach(([benchmarkId, benchmarkCases]) => {
> ```

Oh no, must have forgotten to run the test, will fix it ASAP

**Comment by GervinFung** [comment]
> Thanks, @GervinFung for applying the fixes. Looks good.
> 
> Also, check your email, I sent you one a few days ago.

Thanks for merging it! Yeah I will check my email
Also, I saw the benchmark failed

**Pull request merged**